File: apps/problems/factorial/boilerplate-full/function.rs
use std::fs::read_to_string;
use std::io::{self};
use std::str::Lines;

##USER_CODE_HERE##

fn main() -> io::Result<()> {
  let input = read_to_string("/dev/problems/find-factorial/tests/inputs/##INPUT_FILE_INDEX##.txt")?;
  let mut lines = input.lines();
  let num: i32 = lines.next().unwrap().parse().unwrap();
  let result = factorial(num);
  println!("{}", result);
  Ok(())
}

File: apps/problems/array-sum/boilerplate-full/function.js
##USER_CODE_HERE##

const input = require('fs').readFileSync('/dev/problems/array-sum/tests/inputs/##INPUT_FILE_INDEX##.txt', 'utf8').trim().split('\n').join(' ').split(' ');
const size_arr = parseInt(input.shift());
const arr = input.splice(0, size_arr).map(Number);
const result = arraySum(arr);
console.log(result);
    